2020-03-04-all-hands-meeting.md

```diff
@@ -0,0 +1,194 @@
+    - Emails up and running
+
+### Project Updates
+
+- StART
+    - pc:
+        - We had our combined review/retro/planning, went well though dc was unavailable
+        - First demo with features to show!
+        - Seems c4c felt good even though things aren't perfect
+        - New "definition of done" criteria to ensure all new features have usage tracking
+- Quartermaster
+    - uv:
+        - thurs/fri next week signing. had one paid session, but no contract yet.
+        - pushing invoice cycle out a month. team ok not being paid right away.
+        - had first session. dc and uv came prepped with 20 project write-ups.
+            - used as way to talk about different directions he could take. client really liked this.
+        - project = currently very open-ended and ambiguous.
+        - offering will likely be lots of workshopping.
+        - next up: product walkthrough. show data. share issues.
+- Aether
+    - bl:
+        - now getting user feedback. better idea of market positioning. dev can now be guided by immediate user needs. use of jira board with task priorities bl can pick up.
+
+### Announcements
+
+- Next week, March 11, 30 minutes to discuss dc's proposal to start her research!
+    - [Gdrive with Academic Proposal and Ethics](https://drive.google.com/drive/folders/1GKAPRigk-nj_IZMxjFPathjsxIc5iimh) :lock: 
+    - [Project Partnership Proposal (Totally a Pitch!)](/kLoRWrsnQAGg29pVtZvYSQ)
+        - Preliminary consensus to go-ahead back in Summer 2019 (phew!)
+        - Since then: dc has done desk research on history around alternative networks and decentralization. Profiled/identified contemporary dweb projects, about to approach other sites.
+        - Would want to discuss and revise this together. Seeking to have 1 week review after call, tentatively Mar 18 to decide?
+
+- bl: Use **March 11** meeting discussion time for dc to discuss start of research? (+1 from all)
+
+### Discussions
+
+- Approvals needed today (10 min)
+    - Biz: [Intro deck](https://docs.google.com/presentation/d/1y8Mnyw147rsX1n5NsDek6dJHl9PWU3cwG-_QXBMfY98/edit#slide=id.g7d108645b5_0_80)
+        - Choose whichever version of graphics you want to send based on existing options, don't have to agree on a specific one
+    - Biz: [Starling](https://github.com/hyphacoop/organizing-private/pull/25)
+        - pc: excited. trust guardian project. bl: heard good things about witness project. Starling project wants to tell a use story for dweb tech. all: no blocks.
+    - Fin: [Public holiday policy](https://github.com/hyphacoop/organizing/issues/156#issuecomment-591363595)
+        - Rough wording:
+            - >The Co-operative recommends Members to not work on a public holiday, but in the case where Members voluntarily chooses to or under special circumstances, they will receive regular wages and "substitute time off", and there should be no expectation of "public holiday plus premium pay".
```

Comment:
Are we moving this somewhere? Do we wish to have a "policy" or "agreements" section of our handbook for working groups to write this stuff down? (with maybe future migration into bylaws) Or is everything we write "policy", and agreed-upon things just kinda co-mingle with stuff individuals might write? 

Also, if we record it, I'd suggest we consider a way of including an "expiration" or "term end" after which it gets reviewed.

Reply:
This is to keep record of what we agreed on, I have since modified the text a bit to read better, and included it in handbook.

Reply:
I actually _don't_ think operational recommendations like this belong as a _recommendation_ in the handbook. Fine for now, but I think we should revise

Reply:
Future problem :)